### PR TITLE
Fix Upstream Testing

### PR DESCRIPTION
**Pull request by smk4664**
By removing the config option for makemigrations.

This also brings the app up to the LTM 1.6 Cookiecutter standard and resolves an issue with Griffe being unable to generate the docs for the serializers.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,186 +1,236 @@
...
-on:  # yamllint disable-line rule:truthy
-  - "push"
-  - "pull_request"
+concurrency:  # Cancel any existing runs of this workflow for this same PR
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+on: # yamllint disable-line rule:truthy rule:comments
+  push:
+    branches:
+      - "main"
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:9]: Wouldn't this cause CI to stop running on ltm PRs after this is merged

Comment (smk4664) [reply .github/workflows/ci.yml:9]: Isn't that what the following handles?
```
  pull_request: ~
```

Comment (smk4664) [reply .github/workflows/ci.yml:9]: Ah, I think this would prevent releases from the ltm-1.6 branch, updating.

Comment (gsnider2195) [reply .github/workflows/ci.yml:9]: That handles the checks in the PR but after it's merged into main, ltm or develop we run additional checks.

Comment (gsnider2195) [reply .github/workflows/ci.yml:9]: This looks good now though

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -192,24 +243,24 @@ jobs:
       - name: "Upload binaries to release"
         uses: "svenstaro/upload-release-action@v2"
         with:
-          repo_token: "${{ secrets.GH_NAUTOBOT_BOT_TOKEN }}"
+          repo_token: "${{ secrets.NTC_GITHUB_TOKEN }}"  # use GH_NAUTOBOT_BOT_TOKEN for Nautobot Org repos.
```
Comment (gsnider2195) [inline .github/workflows/ci.yml:246]:
```suggestion
          repo_token: "${{ secrets.GH_NAUTOBOT_BOT_TOKEN }}"  # use GH_NAUTOBOT_BOT_TOKEN for Nautobot Org repos.
```

**Review by gsnider2195 — APPROVED** [review]
You weren't kidding about this getting out of hand!

**Pull request merged**